src/views/Correction.vue

```diff
@@ -329,7 +329,7 @@ export default {
       }
     }
     // NB: this is the id of the filing to correct
-    const origFilingId = this.$route.params.id
+    const origFilingId = +this.$route.params.id // number
```

Comment:
What does the prefixed + do again? Cast it to a specific type?

Reply:
It's called the unary plus operator. It tries to convert the operand into a number (or NaN).
https://www.google.com/search?q=javascript+unary+operator

Reply:
ahhh right, thanks :) This is at least the second time i've asked you now, sorry and thanks!